dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/AbstractClouderaManagerTask.java

```diff
@@ -32,10 +37,15 @@ public AbstractClouderaManagerTask(String targetPath) {
   @Override
   protected final Void doRun(TaskRunContext context, @Nonnull ByteSink sink, @Nonnull Handle handle)
```

Comment:
nit: Why are we using ```Void``` in one doRun definition but ```void``` in the other? Is there a reason not to unify? (I probably miss the point of using ```Void``` altogether).

Reply:
I know man, it's Java, it looks ugly. This `Void` is what we inherit from our `Task<T>`, but it's useless in context of Http tasks, so I simplified it for the particular tasks and hide in the abstract class. However it's `public` api so I can't remove or make it private.

Reply:
Ack. Thanks for the explanation :)